docker-compose.yaml

```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
     database:
         image: postgres:13
-        ports: [5432]
+        ports: [49172:5432]
```

Comment:
You should put this in a `docker-compose.override.yaml` instead, to allow other developers to easily use the project on their machine

I usually create a `docker-compose.override.yaml.dist` and put `docker-compose.override.yaml` in gitignore to customize the local setup.